pages/message-sent/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { BaseContainer } from "@components/Page/BaseContainer";
+import { Helmet } from "react-helmet";
+import React from "react";
+import LayoutContainer from "@components/LayoutContainer";
+import Image from "next/image";
+import getConfig from "next/config";
+import { useRouter } from "next/router";
+
+const { publicRuntimeConfig } = getConfig()
+
+export default function MessageSent() {
+    const router = useRouter();
+    return <BaseContainer className="bg-black">
+        <Helmet>
+            <title>Your message has been sent!</title>
+        </Helmet>
+        <LayoutContainer>
+            <div className="h-full flex flex-col justify-between">
+                <div className="flex-1-0-0 relative overflow-hidden rounded-2xl">
+                    <img className="h-full w-full absolute object-cover" alt="verification image" src="/image/DynamicSuccessIllustration.svg" />
+                </div>
+                <div className="pt-4 gap-y-4 flex flex-col h-min">
+                    <text className="font-inter font-medium text-2xl">Your anonymous message has been sent!</text>
+                    <div className="rounded-lg rounded-t-xl flex flex-col">
+                        <div className="flex flex-col text-base font-inter gap-4 mb-4">
+                            <text className="font-inter font-semibold">Get the app to:</text>
+                            <text className="font-inter font-normal">💬 See replies to your messages </text>
+                            <text className="font-inter font-normal">🕵️‍♀️ Get your own anonymous messages! </text>
+                            <text className="font-inter font-normal">🔔 See what’s new in your school or community</text>
+                        </div>
+                        <button onClick={() => router.push(publicRuntimeConfig.DOWNLOAD_BETTERSOCIAL_APP_URL)} className=" w-full bg-cyan hover:bg-cyan10 text-white font-lato flex py-2 px-4 rounded-lg justify-center items-center gap-2 text-justify">
+                            Download BetterSocial Now
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </LayoutContainer>
+    </BaseContainer>
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code looks good overall, but there's a minor issue with the use of `text` elements on lines 23, 26, 27, 28, and 29. In React, we typically use `p`, `span`, or other semantic HTML tags instead of `text`. This is because `text` is not a standard HTML tag and may not be recognized by all browsers or assistive technologies.

Here's how you can replace `text` with `p`:

```diff
- <text className="font-inter font-medium text-2xl">Your anonymous message has been sent!</text>
+ <p className="font-inter font-medium text-2xl">Your anonymous message has been sent!</p>

- <text className="font-inter font-semibold">Get the app to:</text>
+ <p className="font-inter font-semibold">Get the app to:</p>

- <text className="font-inter font-normal">💬 See replies to your messages </text>
+ <p className="font-inter font-normal">💬 See replies to your messages </p>

- <text className="font-inter font-normal">🕵️‍♀️ Get your own anonymous messages! </text>
+ <p className="font-inter font-normal">🕵️‍♀️ Get your own anonymous messages! </p>

- <text className="font-inter font-normal">🔔 See what’s new in your school or community</text>
+ <p className="font-inter font-normal">🔔 See what’s new in your school or community</p>
```

Also, it's a good practice to add an `alt` attribute to the `Image` component for accessibility reasons. If the image fails to load or if a user is using a screen reader, the `alt` attribute provides alternative information. Here's how you can add it:

```diff
- <Image className="h-full w-full absolute object-cover" alt="verification image" src="/image/DynamicSuccessIllustration.svg" />
+ <Image className="h-full w-full absolute object-cover" alt="Verification illustration" src="/image/DynamicSuccessIllustration.svg" />
```

Lastly, there's a missing newline at the end of the file. It's a common convention to end files with a newline because some tools might not handle or display the last line correctly if it doesn't end with a newline.

```diff
- }
+ }
+
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->